src/ui/elements/job-view-record.element.ts

```diff
@@ -23,25 +25,20 @@ export const JobViewRecord = defineElement<{
 
         const entryTitle = inputs.record.contactName || inputs.record.companyName || dateString;
 
-        const tableRows = getObjectTypedEntries(inputs.record)
-            .sort(([a], [b]) => a.localeCompare(b))
-            .map(
-                ([
-                    recordKey,
-                    recordValue,
-                ]) => {
-                    if (!recordValue || !check.isString(recordValue)) {
-                        return nothing;
-                    }
+        const jobRecordKeyOrder: (keyof JobSearchRecord)[] = getObjectTypedKeys(
```

Comment:
nit: since this is using `getObjectTypedKeys`, the explicit type isn't necessary:
```suggestion
        const jobRecordKeyOrder = getObjectTypedKeys(
```

Reply:
Fixed.